{"uid":"35049f37bcc2718","name":"test_successful_login","fullName":"tests.test_login#test_successful_login","historyId":"f4e42ed032845b0ed5c71539b726bc6e","time":{"start":1701684847238,"stop":1701684863501,"duration":16263},"status":"failed","statusMessage":"AssertionError: Element with locator ('id', 'flash-popup') not visible within 10 seconds.","statusTrace":"self = <pages.login_page.LoginPage object at 0x7fb01f08da60>, locator = ('id', 'flash-popup')\ntimeout = 10\n\n    def wait_for_element_to_be_visible(self, locator, timeout):\n        try:\n>           WebDriverWait(self.driver, timeout).until(EC.visibility_of_element_located(locator))\n\npages/base_page.py:15: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"67f5f09e461d5f26ef0fc6e2cc0fa8d4\")>\nmethod = <function visibility_of_element_located.<locals>._predicate at 0x7fb01f074d60>, message = ''\n\n    def until(self, method, message: str = \"\"):\n        \"\"\"Calls the method provided with the driver as an argument until the \\\n        return value does not evaluate to ``False``.\n    \n        :param method: callable(WebDriver)\n        :param message: optional message for :exc:`TimeoutException`\n        :returns: the result of the last call to `method`\n        :raises: :exc:`selenium.common.exceptions.TimeoutException` if timeout occurs\n        \"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.monotonic() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, \"screen\", None)\n                stacktrace = getattr(exc, \"stacktrace\", None)\n            time.sleep(self._poll)\n            if time.monotonic() > end_time:\n                break\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message:\n\n../../../anaconda3/envs/automated-testing-framework/lib/python3.12/site-packages/selenium/webdriver/support/wait.py:95: TimeoutException\n\nDuring handling of the above exception, another exception occurred:\n\ndriver = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"67f5f09e461d5f26ef0fc6e2cc0fa8d4\")>\n\n    @pytest.mark.login\n    def test_successful_login(driver):\n        login_page = LoginPage(driver)\n        login_page.open()\n        login_page.login(VALID_EMAIL, VALID_PASSWORD)\n>       login_page.wait_for_element_to_be_visible(login_page.locators.WELCOME_MESSAGE,10)\n\ntests/test_login.py:10: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <pages.login_page.LoginPage object at 0x7fb01f08da60>, locator = ('id', 'flash-popup')\ntimeout = 10\n\n    def wait_for_element_to_be_visible(self, locator, timeout):\n        try:\n            WebDriverWait(self.driver, timeout).until(EC.visibility_of_element_located(locator))\n        except TimeoutException:\n>           raise AssertionError(f\"Element with locator {locator} not visible within {timeout} seconds.\")\nE           AssertionError: Element with locator ('id', 'flash-popup') not visible within 10 seconds.\n\npages/base_page.py:17: AssertionError","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"driver","time":{"start":1701684838531,"stop":1701684847213,"duration":8682},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"afterStages":[{"name":"driver::0","time":{"start":1701684876566,"stop":1701684876992,"duration":426},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"labels":[{"name":"tag","value":"login"},{"name":"parentSuite","value":"tests"},{"name":"suite","value":"test_login"},{"name":"host","value":"hilsCYBER"},{"name":"thread","value":"74784-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.test_login"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[{"name":"Product defects","matchedStatuses":[],"flaky":false}],"tags":["login"]},"source":"35049f37bcc2718.json","parameterValues":[]}